tsconfig.json

```diff
@@ -18,7 +18,9 @@
       "#service/*": ["./src/service/*"],
 
       "#test/*": ["./test/*"],
-      "#testing/*": ["./testing/*"]
+      "#testing/*": ["./testing/*"],
+
+      "#third_party/*": ["./third_party/*"]
```

Comment:
Yes this will technically require a VSCode restart for it to update; no it doesn't matter, since `third_party` imports are rare anyway